containers/kubernetes/how-to/enable-easy-deploy.mdx

```diff
@@ -35,7 +35,7 @@ You can also deploy off-the-shelf applications pre-configured for Scaleway produ
 </Message>
 
 1. Click **Kubernetes** in the **Containers** section of the [Scaleway console](https://console.scaleway.com) side menu. The Kubernetes Kapsule dashboard displays.
-2. Click the name of the cluster you wish to deploy your image on. The Cluster Information tab displays.
+2. Click the name of the cluster you wish to deploy your image on. The **Cluster information** tab displays.
```

Comment:
```suggestion
2. Click the name of the cluster you wish to deploy your image on. The **cluster information** tab displays.
```

Reply:
I'd leave the capital since it refers to the name of the tab?